sim/src/mechanics/intersection.rs

```diff
@@ -430,15 +431,18 @@ impl IntersectionSimState {
             .unwrap_or(false)
         {
             // If we started an uber-turn, then finish it! But alert if we're running a red light.
+            // TODO: Consider reenabling alert
```

Comment:
Same, I'd prefer leaving the `if false && ...`

Reply:
Done.